{{cookiecutter.project_slug}}/config/settings/production.py

```diff
@@ -236,10 +236,13 @@
 # https://django-compressor.readthedocs.io/en/latest/settings/#django.conf.settings.COMPRESS_ENABLED
 COMPRESS_ENABLED = env.bool("COMPRESS_ENABLED", default=True)
 # https://django-compressor.readthedocs.io/en/latest/settings/#django.conf.settings.COMPRESS_STORAGE
-{%- if cookiecutter.cloud_provider == 'AWS' %}
+{%- if cookiecutter.cloud_provider == 'AWS' and cookiecutter.use_whitenoise == 'y' %}
 COMPRESS_STORAGE = "storages.backends.s3boto3.S3Boto3Storage"
```

Comment:
I've tested quickly this combination and it's still broken. I know it's not the one you're trying to fix here, but I'm wondering if this value of `COMPRESS_STORAGE` makes sense.

With Whitenoise, the static files are hosted on the application server, why the compress files should be stored on AWS?

I did a quick test and it looks like we don't need to customise the storage when using whitenoise.

Reply:
Hm I quickly glossed over white noise (never using a CDN before), but I assumed the files were hosted on the instance/server itself rather than the bucket. Taking a look again, are the CDN's files still in these buckets? Or are they just kinda... cached?